app/controllers/sessions_controller.rb

```diff
@@ -6,8 +6,13 @@ def new
 	def create
 		user = User.where(email: params[:email]).first
 		if user && user.authenticate(params[:password])
-			session[:user_id] = user.id
-			redirect_to home_path, notice: "Welcome #{user.full_name}!"
+			if user.active?
```

Comment:
Indentation is really off here, You have to stick to 2 spaces per indentation level. Otherwise this kind of gives bad impression, because ruby community really sticks to this styling rule. Either you don't care much about it (bad echo for potential employer) or it tells me you are beginner. Anyway, always try to read your own code at GitHub.